projects/plugins/jetpack/modules/likes/style.css

```diff
@@ -217,3 +217,9 @@ div.sharedaddy.sd-like-enabled .sd-like .post-likes-widget {
 .sd-gplus .sd-title {
 	display: none;
 }
+
+@media print {
+	.jetpack-likes-widget-wrapper {
+		display: none !important;
```

Comment:
Do we need `!important` here? Is it because `.jetpack-likes-widget-wrapper` has an inline style like in #21939?

Reply:
I have fixed it, @danielpost. Unlike in the link mentioned above, there is no inline property here and the use of `!important` is not necessary.